ajax/libs/medium-editor-insert-plugin/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "medium-editor-insert-plugin",
+  "version": "2.4.1",
+  "filename": "js/medium-editor-insert-plugin.min.js",
+  "description": "jQuery insert plugin for Medium editor",
+  "homepage": "http://linkesch.com/medium-editor-insert-plugin/",
+  "keywords": [
+    "contenteditable",
+    "editor",
+    "medium",
+    "wysiwyg",
+    "rich-text",
+    "images",
+    "embeds",
+    "videos"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/orthes/medium-editor-insert-plugin"
+  },
+  "license": "MIT",
+  "author": "Pavel Linkesch <@orthes> (http://linkesch.com/)",
```

Comment:
@dakshshah96 I think we can follow the format as the same as upstream. ref: https://github.com/orthes/medium-editor-insert-plugin/blob/2.4.1/package.json#L15-L19